src/Symfony/Component/Routing/RouteCollectionBuilder.php

```diff
@@ -61,21 +61,28 @@ public function __construct(LoaderInterface $loader = null)
      */
     public function import($resource, $prefix = '/', $type = null)
```

Comment:
Not about the "leak", as routing resources are tracked in a different file, so we cannot reused the same logic.